### PR TITLE
Fixes #341 Added a-frame registry icon

### LINKED ISSUE #341: Add a link next to "Add component" to the aframe-registry ghpages
It should link to https://aframe.io/aframe-registry/
I tried to use the same logo, but it doesn't fit with the overall aesthetics.
image

Any idea what to use here? Maybe just the `?` icon as with the components but bigger?
/cc @ngokevin @feiss

### PR DESCRIPTION
**Pull request by fernandojsg**
Added a button with a link to https://aframe.io/aframe-registry next to the `Add component` selectbox with the logo designed by @feiss
_Note that the img url is absolute to the aframe-inspector so it won't work until deployed_

image

image

### CODE REVIEW

**Comment by dmarcos** [comment]
This looks good to me. We can refine the icon later.

File: src/components/components/AddComponent.js [hunk 1]
```diff
@@ -99,6 +99,9 @@ export default class AddComponent extends React.Component {
             onChange={this.addComponent}
             searchable={true}
           />
+          <a href="https://aframe.io/aframe-registry" target="_blank" title="A-Frame Registry" className="button aregistry">
+            <img src="https://aframe.io/aframe-inspector/assets/a-registry-logo-min.svg"/>
```
Comment (ngokevin) [inline src/components/components/AddComponent.js]: You could package/require this with Webpack.

**Comment by ngokevin** [comment]
Highlighted mode can be same height as the select bar.

Nice, r+wc

File: src/components/components/AddComponent.js (hunk 1 above)
Comment (fernandojsg) [reply src/components/components/AddComponent.js]: I've no idea how to do that :D any help?

Comment (ngokevin) [reply src/components/components/AddComponent.js]: ok i can do later

Comment (fernandojsg) [reply src/components/components/AddComponent.js]: thanks!

**Comment by fernandojsg** [comment]
@ngokevin done! thank you

**Pull request merged**